common/djangoapps/student/models.py

```diff
@@ -1028,7 +1028,7 @@ class CourseEnrollmentManager(models.Manager):
     Custom manager for CourseEnrollment with Table-level filter methods.
     """
 
-    def num_enrolled_in(self, course_id):
+    def num_enrolled_in(self, course_id, max_enrollments):
```

Comment:
What if `max_enrollments` is not passed? or I would like to get all enrollments?

Reply:
There isn't any other usage of this method. Bringing the count of ALL ENROLLMENTS was actually causing the issue here.

Reply:
I think in that case, the name of the `method` is confusing. I would suggest either
1. Change the method name that align's correct with the need of this method.
2. Introduce a new method for your need and leave the existing method as is. -- as you said there is no other place this makes me lean towards the first option.

Reply:
I've changed the name. 👍